app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/components/ResponseDelivery.tsx

```diff
@@ -245,6 +246,15 @@ export const ResponseDelivery = () => {
     }
 
     updateField("formPurpose", purposeOption);
+    result = (await updateTemplateFormPurpose({
+      id,
+      formPurpose: formPurpose,
+    })) as FormServerError;
+
+    if (result?.error) {
+      toast.error(<ErrorSaving errorCode={FormServerErrorCodes.DELIVERY_OPTION} />, "wide");
```

Comment:
This error should match form purpose vs `DELIVERY_OPTION`

Reply:
For FormServerErrorCodes - do we have an established pattern for these? Should I just increment to 553?

Reply:
yes for now just increment we haven't established patterns for these yet